### PR TITLE
[tests-only][full-ci]Backport search folder by tag in project space

### PR DESCRIPTION
**Pull request by nabim777**
This PR is Backport of master PR https://github.com/owncloud/ocis/pull/6968

### CODE REVIEW

**Review by SwikritiT — APPROVED** [review]

File: tests/acceptance/features/apiGraph/fullSearch.feature [hunk 1]
```diff
@@ -92,6 +92,29 @@ Feature: full text search
...
+    Then the HTTP status code should be "207"
+    And the search result of user "Alice" should contain only these files:
+      | spacesFolder    |
+      | spacesSubFolder |
+    Examples:
+      | dav-path-version |
+      | old              |
+      | new              |
+      | spaces           |
+
```
Comment (SwikritiT) [inline tests/acceptance/features/apiGraph/fullSearch.feature:116]: Seems like this doesn't work for spaces in stable
```feature
    Examples:
      | dav-path-version |
      | old              |
      | new              |
      | spaces           |
        Failed step: And the search result of user "Alice" should contain only these files:
        The number of elements in the response doesn't matches with expected number of elements
        Failed asserting that 0 matches expected 2.
```
```suggestion
```

Comment (nabim777) [reply tests/acceptance/features/apiGraph/fullSearch.feature:116]: ok I will keep the tag `@skipOnStable3.0`

Comment (SagarGi) [reply tests/acceptance/features/apiGraph/fullSearch.feature:120]: @nabim777  does it not work in stable-3.0 or what? it seems to be passing in master isn't it?

Comment (SwikritiT) [reply tests/acceptance/features/apiGraph/fullSearch.feature:120]: Yes I think it works for master. I think it this case we can just remove the `stable` option from the example if it's not supposed to work for stable insted to skipping the whole scenario , but is it supposed to not work for stable? cc @saw-jan

Comment (nabim777) [reply tests/acceptance/features/apiGraph/fullSearch.feature:120]: There was same [issue in master](https://github.com/owncloud/ocis/issues/6669) also later after adding `remote.php` in spaces dav endpoint that works fine and passes. But in here after adding the `remote.php` it gives `405` status

> @nabim777 does it not work in stable-3.0 or what? it seems to be passing in master isn't it?

**Review by SagarGi — APPROVED** [review]

File: tests/acceptance/features/apiGraph/fullSearch.feature [hunk 2]
```diff
@@ -92,6 +92,33 @@ Feature: full text search
...
+    Examples:
+      | dav-path-version |
+      | old              |
+      | new              |
+
+    @skipOnStable3.0
+    Examples:
+      | dav-path-version |
+      | spaces           |
+
```
Comment (SwikritiT) [inline tests/acceptance/features/apiGraph/fullSearch.feature:120]: Note that this exact change should be made in master as well other wise there will be conflict while merge the stable to master. We want as minimal conflicts as possible

**Review by phil-davis — APPROVED** [review]

**Comment by phil-davis** [comment]
@nabim777 please make sure to forward-port the small example-table changes to master.

**Pull request merged**